gnome-extra/gnome-commander/Manifest

```diff
@@ -1,2 +1,4 @@
+DIST gnome-commander-1.10.2.tar.xz 7433468 BLAKE2B a62a39490b154a8dc7a5e05c4d4c59fc79447b7b9ddae139542552f809b7a53cf729d636f1fabb85c6390520a25d84477ec6dfcd4394eefe59a8039b51b0ddf2 SHA512 7a411ca6f604771445119cb2b9d14f7f5d91622d28975e92ae5f5da3fc4f76fe51cca0de3712377ef95064879089c1c723dce9a824e5784cfcd484b869702dad
 DIST gnome-commander-1.10.1.tar.xz 7424500 BLAKE2B 6abddf8dae4bc0a1d740ffee49ccc7d816d783bdb5c2b90c9d0681c1db5fa3f42700e65a780881400c1af038d7dfe7f458a98cf2f8f1c1b0fc6f93a3d58dc069 SHA512 b5bc548444149e00f29e6e3b7f785c99a08d27a0a2642cf78c203411a8d2937341a76685b09172047162afe3e0fc92ff13c82bdbd8105281d360c1df44b4702d
 DIST gnome-commander-1.10.0.tar.xz 7421328 BLAKE2B 4cfb6a5d445217de10007d4371cd5cce95319dd68a6e7f071def2258cd965cfbeab671c6016e624ef474501608214f309c56360216cc53c83a1660b38dbe77d9 SHA512 76e2cca3af7e11ae7f327fa25bb175ad28d36d8b71c4908ce8b91533ed53a444a30d861a24cf94ee34dd7954d1f9dd0a14d26c2214854424b80f8e7c107db60d
+
```

Comment:
please drop that superfluous empty line (looks like manual editing rather than using `repoman` or `ebuild` for generating the manifest?)

Reply:
I normaly use repoman, but here I actually added the line manually and accidentally. ;)